server/src/main.rs

```diff
@@ -33,6 +38,7 @@ use crate::catalog::Catalog;
 use crate::optimizer::model_simple_aggregates;
 use crate::tables::ModelTable;
 
+
```

Comment:
There should only be one newline between imports and the rest of the code.

Reply:
Fixed